fhir-core/src/main/java/com/ibm/fhir/core/HTTPHandlingPreference.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.fhir.core;
+
+/**
+ * An enumeration of handling preference codes.
+ * 
+ * @see https://tools.ietf.org/html/rfc7240#section-4.4
```

Comment:
in the future, we should consider using an a href - check @see form 2 --> https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#CHDJGIJB

> Form 2. The @see <a href="URL#value">label</a> form adds a link as defined by URL#value. The URL#value parameter is a relative or absolute URL. The javadoc command distinguishes this from other cases by searching for a less-than symbol (<) as the first character. For example, @see <a href="spec.html#section">Java Spec</a>

Reply:
```suggestion
 * @see <a href="https://tools.ietf.org/html/rfc7240#section-4.4">href="https://tools.ietf.org/html/rfc7240#section-4.4</a>
```